b2sdk/transfer/emerge/executor.py

```diff
@@ -370,8 +371,9 @@ def _match_unfinished_file_if_possible(
             # FIXME: what if `encryption is None` - match ANY encryption? :)
             if encryption is not None and encryption != file_.encryption:
                 continue
-            if bool(legal_hold) != file_.legal_hold:
-                # when `file_.legal_hold is None` it means that `legal_hold` is unknown and we skip
+            if (legal_hold or LegalHold.UNSET) != file_.legal_hold:
```

Comment:
```
>>> (LegalHold.OFF or LegalHold.UNSET) != LegalHold.OFF
True

```

Reply:
why would we compare only enum values and not the one in the argument and the one in the file version?